content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -98,7 +98,7 @@ spec:
 status:
   validationMessages:
   - code: IST0101
-    documentation_url: https://istio.io/docs/reference/config/analysis/IST0101?ref=status-controller
+    documentation_url: https://istio.io/docs/reference/config/analysis/ist0101?ref=status-controller
```

Comment:
```suggestion
    documentation_url: https://istio.io/docs/reference/config/analysis/ist0101/?ref=status-controller
```

Reply:
Thanks @frankbu . Hard to find the missing `/` in the output.